### PR TITLE
make CI logging installation work with latest 4.1

### PR DESCRIPTION
**Pull request by richm**

### CODE REVIEW

**Comment by richm** [comment]
now ready to review - should unblock CI

File: openshift/ci-operator/build-image/elasticsearch-subscription.yaml [hunk 1]
```diff
@@ -7,5 +7,4 @@ spec:
   source: elasticsearch
   sourceNamespace: openshift-operators
   name: elasticsearch-operator
-  startingCSV: elasticsearch-operator.v0.0.1
   channel: preview
```
Comment (nhosoi) [inline openshift/ci-operator/build-image/elasticsearch-subscription.yaml]: Is it ok having v1alpha1 intact?
`elasticsearch-subscription.yaml:apiVersion: operators.coreos.com/v1alpha1`

The same question for cluster-logging-subscription.yaml...
`cluster-logging-subscription.yaml:apiVersion: operators.coreos.com/v1alpha1`

Comment (richm) [reply openshift/ci-operator/build-image/elasticsearch-subscription.yaml]: > Is it ok having v1alpha1 intact?
> `elasticsearch-subscription.yaml:apiVersion: operators.coreos.com/v1alpha1`
> 
> The same question for cluster-logging-subscription.yaml...
> `cluster-logging-subscription.yaml:apiVersion: operators.coreos.com/v1alpha1`

Yes.  For some reason, the subscription api is still v1alpha1

Comment (bparees) [reply openshift/ci-operator/build-image/elasticsearch-subscription.yaml]: i would assume yes, i would have expected that in conjunction w/ the marketplace api changes to v1.  

@ecordell @jwforres can you weigh in?

Comment (richm) [reply openshift/ci-operator/build-image/elasticsearch-subscription.yaml]: If so, we'll have to open another PR, if/when that happens . . .

**Pull request merged**